sotodlib/site_pipeline/make_det_info_wafer.py

```diff
@@ -6,32 +6,69 @@
 from argparse import ArgumentParser
 
 from detmap.makemap import MapMaker
+import sotodlib
 from sotodlib import core
 from sotodlib.io.metadata import write_dataset
 
 logger = logging.getLogger(__name__)
 
-
 def parse_args():
     parser = ArgumentParser()
-    parser.add_argument('-c', '--config-file', help="Configuration File for running DetMap")
+    parser.add_argument('-c', '--config-file', 
+        help="Configuration File for running DetMap")
+    parser.add_argument('--overwrite', action='store_true', 
+        help="Overwrite existing entries.")
+    parser.add_argument('-v', '--verbose', action='count',
+                        default=0, help="Pass multiple times to increase.")
+    parser.add_argument('--debug', action='store_true')
+    parser.add_argument('--log-file', default='make_det_info_wafer.log', help=
+    "Output log filename")
     args = parser.parse_args()
     return args
 
 
 def main(args=None):
     if args is None:
         args = parse_args()
+        
+    if args.debug:
+        logger.setLevel(logging.INFO)
```

Comment:
Instead of formatting the logger here, could you start with the site-pipeline logger?  (we have multiple, none are documented) https://github.com/simonsobs/sotodlib/blob/8c35f8bfd50cfe168b6527be31cc07b857316c2c/sotodlib/site_pipeline/util.py#L245 

Also, it's cool to add the log file but could you make it optional? I suspect in the end we'll be capturing stdout and stderr for logs at the site so we might not want it there.